backend/app/utils/file.py

```
@@ -0,0 +1,70 @@
+    return extension
+
+
+def prepare_image_messages(files: List[UploadFile]) -> List[dict]:
+    """Convert uploaded files to the format required by the LLM."""
+    image_messages = []
+    allowed_types = ["image/jpeg", "image/png"]
+
+    for file in files:
+        mime_type = file.content_type or "image/jpeg"
+        if mime_type in allowed_types:
+            image_base64 = base64.b64encode(file.file.read()).decode("utf-8")
```

Ruff (0.14.13)

Undefined name `base64`

(F821)